.github/workflows/ci.yml

```diff
@@ -18,10 +18,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install Rust
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
```

Comment:
I was confused at first why this uses `@master`, but I see the [docs](https://github.com/dtolnay/rust-toolchain?tab=readme-ov-file#inputs) say:

>  When passing an explicit toolchain as an input instead of @rev, you'll want to use "dtolnay/rust-toolchain@master" as the revision of the action.